process/smartContract/processorV2/processV2.go

```diff
@@ -1033,91 +1076,124 @@ func (sc *scProcessor) resolveBuiltInFunctions(
 	return vmOutput, nil
 }
 
+type sameShardExecutionDataAfterBuiltIn struct {
+	isSCCallSelfShard    bool
+	callType             vmData.CallType
+	parsedTransfer       *vmcommon.ParsedESDTTransfers
+	scExecuteOutTransfer *vmcommon.OutputTransfer
+	err                  error
```

Comment:
I think err should no be inside the struct but separately returned from the method below

Reply:
you're right